client/src/components/deals.jsx

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 import Deal from './deal';
 
-const Deals = (props) => (
-  props.deals.map((deal, i) => {
-    const key = i + 1;
-    return (
-      <div key={key}>
+const Deals = ({ deals, onChange, selectedOption, onSale }) => (
+  <form>
+    {deals.map((deal, i) => {
+      const key = i + 1;
```

Comment:
Is the `i + 1` necessary? Is your code relying on this key property anywhere else?

Reply:
I just didn't want a 0 based key, but no other reason than that.

Reply:
Was wondering this too. Could the unique deal title be used for key?